### PR TITLE
Add a "DO NOT EDIT" header to all declaration files.

### PR DESCRIPTION
**Pull request by aomarks**

### CODE REVIEW

**Review by rictic — APPROVED** [review]

File: src/test/goldens/paper-behaviors/paper-button-behavior.d.ts [hunk 1]
```diff
@@ -1,3 +1,10 @@
+/**
+ * DO NOT EDIT
+ *
+ * This file was automatically generated by
+ * https://github.com/Polymer/gen-typescript-declarations.
+ */
```
Comment (rictic) [inline src/test/goldens/paper-behaviors/paper-button-behavior.d.ts]: We should mention the source file that the user should edit, if they want to fix the generated typings.

Comment (aomarks) [reply src/test/goldens/paper-behaviors/paper-button-behavior.d.ts]: You mean the specific source files that it was derived from?

Comment (rictic) [reply src/test/goldens/paper-behaviors/paper-button-behavior.d.ts]: Yeah, like this would mention paper-button-behavior.html (or .js or whatevs)

Comment (aomarks) [reply src/test/goldens/paper-behaviors/paper-button-behavior.d.ts]: Done! PTAL?

File: src/gen-ts.ts [hunk 2]
```diff
@@ -99,6 +101,16 @@ function makeDeclarationsFilename(sourceUrl: string): string {
   return path.join(parsed.dir, parsed.name) + '.d.ts';
 }
 
+function makeHeader(sourceUrls: string[]): string {
+  return `DO NOT EDIT
+
+This file was automatically generated by
+  https://github.com/Polymer/gen-typescript-declarations
+
+Source files:
```
Comment (rictic) [inline src/gen-ts.ts]: I'd recommend:

`To change these typings you'll have to edit the source files:`

**Review by rictic — APPROVED** [review]

File: src/gen-ts.ts (hunk 2 above)
Comment (aomarks) [reply src/gen-ts.ts]: Done.

**Pull request merged**